android/src/main/java/com/rnim/rn/audio/AudioRecorderManager.java

```diff
@@ -170,9 +170,16 @@ public void stopRecording(Promise promise){
       promise.reject("INVALID_STATE", "Please call startRecording before stopping recording");
       return;
     }
-    recorder.stop();
+
+    try {
+      recorder.stop();
+      recorder.release();
+      recorder = null;
+    }
+    catch(final Exception e) {
+      ecorder = null;
```

Comment:
@jsierles it's been a few weeks so I made the changes here: https://github.com/jsierles/react-native-audio/pull/148